python-cache-demo/geo-service/geolocation_db_conn.py

```diff
@@ -0,0 +1,25 @@
+import mysql.connector
+import logging
+
+class GeolocationDBConnector:
+    def __init__(self, host, user, pwd, database="geolocation", logger=logging.Logger(__name__)):
+        self.logger = logger
+        self.logger.info(f"Connecting to the database {database} on the host: {host}.")
+        try:
+            self.db_conn = mysql.connector.connect(
+                host=host,
```

Comment:
These are passed as ENV's (should be passed to geo_service.py when instantiating the DB Connector), also an example is shown in the infra-service

Reply:
Those params are passed in each <name>_service.py separately when establishing the connection.